packages/lesmis/src/components/DashboardExportModal/components/PreviewPage.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { FlexColumn } from '@tupaia/ui-components';
+
+import Header from './Header';
+
+const PreviewContainer = styled.div`
+  margin-top: 20px;
+  min-width: 900px;
+`;
+
+const Content = styled(FlexColumn)`
+  margin: 0px 150px;
+`;
+
+export const PreviewPage = ({ children, getNextPage, currentPage, isExporting, ...configs }) => {
```

Comment:
Yeah I'm not sure we can do preview like this anymore. Since we have a binary PDF now, the only way to render it would be to use a special iframe of some sort that has src of your /api/v1/pdf

Reply:
Discussed. PDF generation takes a while, so iframe solution is not good. This way is better.